packages/runtime-dom/types/jsx.d.ts

```diff
@@ -234,13 +234,14 @@ interface AriaAttributes {
   'aria-valuetext'?: string
 }
 
-// Vue's style normalization supports nested arrays
+// Vue's style and class normalization supports nested arrays
 export type StyleValue = string | CSSProperties | Array<StyleValue>
+export type ClassValue = unknown | Record<string, unknown> | Array<ClassValue>
```

Comment:
`unknown` is used because user can use shortcircuit e.g `maybeTruthy && 'my-class'`